index.html

```diff
@@ -27,43 +35,44 @@
       <div class="header-bar">
         <div class="logo">
           <a href="./index.html"
-            ><div class="sportslogo">
+            ><div class="logopro">
             <i class="fa-solid fa-baseball"></i>
-            <p>Sports</p>
+            <p>SPORTS </p>
+            <!-- <p>IG</p> -->
           </div></a
           >
           <div class="closeBtn"><i class="fas fa-times fa-2x"></i></div>
         </div>
         <nav>
           <ul class="nav-ul">
-            <li><a href="./about.html">About</a></li>
-            <li><a href="#program">Program</a></li>
-            <li><a href="#">Join</a></li>
-            <li><a href="#partner">Sponsors</a></li>
-            <li><a href="#">News</a></li>
+            <li><a class="menu-link" href="./about.html">About</a></li>
+            <li><a class="menu-link" href="#program">Program</a></li>
+            <li><a class="menu-link" href="#">Join</a></li>
+            <li><a class="menu-link" href="#partner">Sponsors</a></li>
+            <li><a class="menu-link" href="#">News</a></li>
           </ul>
         </nav>
         <button class="campaign">Campaign</button>
       </div>
     </header>
     <main>
-      <section class="introduction">
+      <section class="intro">
         <div class="menu">
           <i class="fas fa-bars fa-2x"></i>
         </div>
         <h2>"Hello! Sharing world"</h2>
         <h1>WORLD ANNUAL SPORTS CONFERENCE</h1>
-        <div class="intro-description">
+        <div class="intro-descr">
           <p>
             This page describes the World annual sports conference 
             which holds annually. The aim of the conference is to
             celebrate the impact of the sports in bringing different cultures 
             and people together and also to discuss ways to move forward.
           </p>
         </div>
-        <div class="date">
+        <div class="date-local">
           <h3>2022.03.07(MON) ~ 11(FRI)</h3>
-          <h4> Estadio Santiago Bernabeu, Madrid, Spain</h4>
+          <h4>Estadio Santiago Bernabeu, Madrid, Spain</h4>
         </div>
       </section>
       <section class="main-program" id="program">
```

Comment:
- [ ] I recommend using grid to align content in `main program section`. Because in smaller desktop views (starting from 1081px) the content is wrapping 

Current layout  | Layout with grid
------------- | -------------
<img width="793" alt="Screenshot 2022-03-10 at 13 33 07" src="https://user-images.githubusercontent.com/77038610/157633026-ca554845-845f-4e4f-8263-2db7de51899f.png"> | <img width="790" alt="Screenshot 2022-03-10 at 13 37 11" src="https://user-images.githubusercontent.com/77038610/157633832-98d808c7-9475-4d1f-989e-6ad97971023e.png">


Here is what I did:
```
.programs {
    display: grid;
    grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
}
```